tensorboard/webapp/notification_center/_views/notification_center_test.ts

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {CommonModule} from '@angular/common';
+import {TestBed} from '@angular/core/testing';
+import {MatButtonModule} from '@angular/material/button';
+import {MatMenuModule} from '@angular/material/menu';
+import {By} from '@angular/platform-browser';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {MatIconTestingModule} from '../../testing/mat_icon_module';
+import {NotificationCenterComponent} from './notification_center_component';
+import {NotificationCenterContainer} from './notification_center_container';
+
+describe('alert snackbar', () => {
```

Comment:
please update this test name; it should be sensible on the command line when all specs run. e.g., `'notification_center/_views/notification_center'`